docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+
+- [Batch Request tutorial](/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset)
+
+## Expectations
+
+An Expectation is a verifiable assertion about Source Data.  Similar to assertions in traditional Python unit tests, Expectations provide a flexible, declarative language for describing expected behaviors. Unlike traditional unit tests which describe the expected behavior of code given a specific input, Expectations apply to the input data itself. For example, you can define an Expectation that a column contains no null values. When GX runs that Expectation on your data it generates a report which indicates if a null value was found.
+
+Expectations improve quality for data applications by giving you a verifiable standard to measure your data against.  They also enhance communication about your data by giving you a source of truth that describes the state it should conform to. Expectations help you take the implicit assumptions about your data and make them explicit and shareable, reducing the need to consult subject matter experts about uncertainties and let you avoid leaving insights about data in isolated silos.
+
+Expectations can be built directly from the domain knowledge of subject matter experts, interactively while introspecting a set of data, or through automated tools provided by GX.
+
+For a list of available Expectations, see [the Expectation Gallery](https://greatexpectations.io/expectations/).
+
+
+- [Expectation Tutorials](/docs/guides/expectations/expectations_lp)
+
+### Expectation Suites
+
+Expectation Suites are collections of Expectations describing your data.  When GX validates data, an Expectation Suite helps streamline the process by running all of the contained Expectations against that data.  In almost all cases, when you create an Expectation you will be creating it inside of an Expectation Suite object.
+
+You can define multiple Expectation Suites for the same data to cover different use cases.  An example could be having one Expectation Suite for raw data, and a more strict Expectation Suite for that same data post-processing.  Because an Expectation Suite is decoupled from a specific source of data, you can apply the same Expectation Suite against multiple, disparate Datasources.  For instance, you can reuse an Expectation Suite that was created around an older set of data to validate the quality of a new set of data.
+
+- [Expectations and Expectation Suite tutorials](/docs/guides/expectations/create_manage_expectations_lp)
+
+### Data Assistants
+
+A Data Assistant is a utility that automates the process of building Expectations by asking questions about your data, gathering information to describe what is observed, and then presenting Metrics and proposed Expectations based on the answers it has determined.  This can greatly accelerate the process of creating Expectations for the provided data.
```

Comment:
```suggestion
A Data Assistant is a utility that automates the process of building Expectations by asking questions about your data, gathering information to describe what is observed, and then presenting Metrics and proposed Expectations based on the answers it has determined.  This can accelerate the process of creating Expectations for the provided data.
```

Reply:
suggestion implemented; it'll be in the next commit.